<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1369</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestQT1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(88, 88, 88);
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSpinBox" name="spinBox1"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox2"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labelIn">
          <property name="minimumSize">
           <size>
            <width>640</width>
            <height>480</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>640</width>
            <height>480</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelOut">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>640</width>
            <height>480</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>640</width>
            <height>480</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(200, 200, 200);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="sideMenu"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1369</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
   </widget>
   <widget class="QMenu" name="menuA_adir">
    <property name="title">
     <string>AÃ±adir</string>
    </property>
    <addaction name="actionConversion"/>
    <addaction name="actionThresh"/>
    <addaction name="actionFilter"/>
    <addaction name="actionConnected"/>
    <addaction name="actionChannel"/>
   </widget>
   <widget class="QMenu" name="menuPipelines">
    <property name="title">
     <string>PIPELINES</string>
    </property>
    <addaction name="actionSegmentation"/>
   </widget>
   <widget class="QMenu" name="menuOutFiles">
    <property name="title">
     <string>OUT FILES</string>
    </property>
    <addaction name="actionDispersion"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionCamera"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuA_adir"/>
   <addaction name="menuPipelines"/>
   <addaction name="menuOutFiles"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionREC"/>
   <addaction name="actionCAP"/>
   <addaction name="actionRES"/>
   <addaction name="actionUPD"/>
   <addaction name="actionPRO"/>
   <addaction name="actionTrain"/>
   <addaction name="actionTEST"/>
  </widget>
  <action name="actionAbrir">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionMIN_MAX">
   <property name="text">
    <string>MIN/MAX</string>
   </property>
  </action>
  <action name="actionThresh">
   <property name="text">
    <string>THRESH</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="text">
    <string>FILTER</string>
   </property>
  </action>
  <action name="actionREC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>REC</string>
   </property>
  </action>
  <action name="actionRES">
   <property name="text">
    <string>RES</string>
   </property>
  </action>
  <action name="actionPRO">
   <property name="text">
    <string>PRO</string>
   </property>
  </action>
  <action name="actionConversion">
   <property name="text">
    <string>CONVERSION</string>
   </property>
  </action>
  <action name="actionUPD">
   <property name="text">
    <string>UPD</string>
   </property>
  </action>
  <action name="actionConnected">
   <property name="text">
    <string>CONNECTED</string>
   </property>
  </action>
  <action name="actionTEST">
   <property name="text">
    <string>TEST</string>
   </property>
   <property name="toolTip">
    <string>TEST</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>&lt;-</string>
   </property>
  </action>
  <action name="actionCount">
   <property name="text">
    <string>0</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>-&gt;</string>
   </property>
  </action>
  <action name="actionChannel">
   <property name="text">
    <string>CHANNEL</string>
   </property>
  </action>
  <action name="actionSegmentation">
   <property name="text">
    <string>SEGMENTATION</string>
   </property>
  </action>
  <action name="actionDispersion">
   <property name="text">
    <string>Dispersion</string>
   </property>
  </action>
  <action name="actionTrain">
   <property name="text">
    <string>TRAIN</string>
   </property>
  </action>
  <action name="actionCAP">
   <property name="text">
    <string>CAP</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TestQT1.qrc"/>
 </resources>
 <connections/>
</ui>
